tasks/tiler/us.py

```diff
@@ -0,0 +1,50 @@
+from tasks.us.census.tiger import ShorelineClip
+from tasks.tiler.xyz import TilerXYZTableTask
+from luigi import WrapperTask
+from lib.logger import get_logger
+
+LOGGER = get_logger(__name__)
+
+GEOGRAPHY_LEVELS = {
+    'state': 'us.census.tiger.state',
+    'county': 'us.census.tiger.county',
+    'census_tract': 'us.census.tiger.census_tract',
+    'block_group': 'us.census.tiger.block_group',
+    'block': 'us.census.tiger.block'
+}
+
+
+class XYZTables(TilerXYZTableTask):
+
+    def __init__(self, *args, **kwargs):
+        super(XYZTables, self).__init__(*args, **kwargs)
+
+    def requires(self):
+        return {
+            'shorelineclip': ShorelineClip(geography=self.geography, year='2015')
+        }
+
+    def get_config_file(self):
+        return 'us_all.json'
+
+    def get_geography_level(self, level):
+        return GEOGRAPHY_LEVELS[level]
+
+
+class AllUSXYZTables(WrapperTask):
+
+    def requires(self):
+        for zoom in range(0, 5):
+            yield XYZTables(zoom_level=zoom, geography=self._get_geography_level(zoom))
+
+    def _get_geography_level(self, zoom):
```

Comment:
I'd replace these conditionals with a `geography_level_by_zoom` dict or array. What do you think?

Reply:
that is wors for flexibility. At first, I did that but I ended having a custom dictionary for every subclass which is worse to implement in the future

Reply:
Oh I didn't see is the other function sorry. Yes, I tried to do because is a range of integers you end having a function to gather the data or having an array with 14 elements. SUggestions?

Reply:
I'd say that an array is a good fit, because it has implicit semantics about key type and failure if zoom is not supported. It might look ugly but it's precise, clear, small and provides O(1) reads. I based my suggestion on a "Writing idiomatic Python" book recommendation, but as there are zoom ranges, the improvement is not _that_ clear. Anyway, it's really minor, if you prefer the `elif` approach, it's fine 👍